docs/enterprise-guide/authenticating-with-saml.md

```diff
@@ -140,6 +140,31 @@ The settings that Metabase defaults to here might work for you out of the box, b
 
 Each of these input boxes needs a URI that points to the location of a SAML attribute.
 
+### Group Schema
+
+The group schema setting allows you to set Metabase groups based on an attribute of your user in your SAML provider. Please note that this may not correlate to group functionality provided by your SAML provider - you may need to create a separate attribute on your users.
```

Comment:
I'm not sure I follow this "Please note that this may not correlate to group functionality provided by your SAML provider - you may need to create a separate attribute on your users."

Reply:
Hmm, let me think about the wording. Basically the idea is that Okta (and probably other SAML providers) have their own idea of what a "group" is, but we don't use those. When I tried to set it up I got super confused and spent a lot of time trying to figure out how to make it work with Okta's group functionality, so I wanted to put some kind of note here.

Reply:
@mazameli Do you think "The group schema setting allows you to set Metabase groups based on an attribute of your user in your SAML provider. Note that this is separate from configuring Groups within your SAML provider, and you will likely need to create a custom attribute on your users" is more clear?

Reply:
I think what I'm confused by is this part: "you will likely need to create a custom attribute on your users." Do you mean create a custom `group` attribute? E.g., "…you will likely need to create a custom attribute on your users to set their Metabase group, like `metabase_group`."

Reply:
Ah, that makes sense, I'll update it.